docs/business.yaml

```diff
@@ -1498,6 +1503,27 @@ paths:
                       legalType: BC
                     noticeOfWithdrawal:
                       filingId: 123456
+              notice-of-withdrawal-with-options-request:
+                summary: Notice of Withdrawal Request with Options
+                value:
+                  filing:
+                    header:
+                      name: noticeOfWithdrawal
+                      certifiedBy: Sample Certified Person
+                      email: api.specs@example.com
+                      date: '2024-12-18'
+                      priority: false
+                    business:
+                      foundingDate: '2023-07-12T17:31:58.000+00:00'
+                      identifier: BC1234567
+                      legalName: 1234567 B.C. LTD.
+                      legalType: BC
+                    noticeOfWithdrawal:
+                      filingId: 123456
+                      courtOrder:
+                        fileNumber: "A12345"
+                        effectOfOrder: planOfArrangement
+                        effectOfArrangement: ''
```

Comment:
We aren't using `effectOfArrangement` and it isn't a court order property anyway. Remove it.

Ref: https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/court_order.json

Reply:
updated